### ISSUE TITLE
"any" is not exported by the List::Util module

### ISSUE THREAD

**Issue by eserte** [vti/sockjs-perl#13]
On a CentOS7 system with the system perl the test suite fails:
```
...
"any" is not exported by the List::Util module
Can't continue after import errors at /home/cpansand/.cpan/build/2018120210/SockJS-0.08-DX7S4T/blib/lib/SockJS/Transport/Base.pm line 6.
BEGIN failed--compilation aborted at /home/cpansand/.cpan/build/2018120210/SockJS-0.08-DX7S4T/blib/lib/SockJS/Transport/Base.pm line 6.
Compilation failed in require at t/transport/base.t line 6.
BEGIN failed--compilation aborted at t/transport/base.t line 6.
t/transport/base.t ........... 
Dubious, test returned 255 (wstat 65280, 0xff00)
No subtests run 
... (etc) ...
```
On this system List::Util 1.27 is installed.

**Comment by vti** [comment]
Should be fixed in 0.09.

**Comment by eserte** [comment]
Yes, looks quite green now.
